static/templates/settings/display_settings.hbs

```diff
@@ -73,6 +73,14 @@
           label=settings_label.translate_emoticons
           prefix=prefix}}
 
+        {{#if page_params.development_environment }}
+        {{> settings_checkbox
+          setting_name="display_emoji_reaction_users"
+          is_checked=settings_object.display_emoji_reaction_users
+          label=settings_label.display_emoji_reaction_users
+          prefix=prefix}}
+        {{/if}}
```

Comment:
I'm removing the `development_environment` guard, since I think we're planning to complete merging the full feature at this point.